### PR TITLE
JAMES-3253 Remove Log warning for remoteDelivery configuration in def…

### PR DESCRIPTION
**Pull request by chibenwa**
…aultMailetContainer.xml for JPA Guice

This file is relied upon when configuration file is missing

### CODE REVIEW

File: server/container/guice/jpa-guice/src/main/resources/defaultMailetContainer.xml [hunk 1]
```diff
@@ -49,7 +49,6 @@
         <mailet match="SMTPAuthSuccessful" class="RemoteDelivery">
             <outgoingQueue>outgoing</outgoingQueue>
             <delayTime>5000, 100000, 500000</delayTime>
-            <maxRetries>25</maxRetries>
```
Comment (rouazana) [inline server/container/guice/jpa-guice/src/main/resources/defaultMailetContainer.xml:52]: not `3` like in most configuration files?

Comment (chibenwa) [reply server/container/guice/jpa-guice/src/main/resources/defaultMailetContainer.xml:52]: Not mentioning it achieve the same.

Comment (rouazana) [reply server/container/guice/jpa-guice/src/main/resources/defaultMailetContainer.xml:52]: and so, any reason to not be coherent between our configuration files?

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**